sormas-ui/src/main/java/de/symeda/sormas/ui/caze/AbstractCaseView.java

```diff
@@ -199,4 +199,11 @@ public ViewMode getViewMode() {
 
 		return viewConfiguration.getViewMode();
 	}
+	
+	public void setCaseEditPermission(Component component) {
+		Boolean isCaseEditAllowed  = FacadeProvider.getCaseFacade().isCaseEditAllowed(getCaseRef().getUuid());
+		if (!isCaseEditAllowed){
+    		getComponent(getComponentIndex(component)).setEnabled(false);
```

Comment:
Is component.setEnabled not working?

Reply:
setEnabled requires a boolean param

Reply:
I meant `component` instead of `getComponent(getComponentIndex(component))`

Reply:
changed and tested